### ISSUE TITLE
Accessibility :MAS13: Zoom: Toggle button is not accessible when zoomed for 200%.

### ISSUE THREAD

**Issue by wiproaccess** [wiproaccess/vscode-website#22]
#A11yMAS
#MAS13
#Wipro
#vscode-website

**Testing Environment:**
OS-RS1
OS Name Microsoft -Windows 10 Pro
Browser-Edge
Edge Version-Microsoft Edge 38.14393.461.0

**Repro Steps:**
1.Open https://code.visualstudio.com/ in edge ,zoom the page to 200%.
2. Navigate to "Visual Studio Code" on the blue label header in top of the page. 
3. Observe the toggle button accessibility with the Keyboard.[screenshot]

**Actual Result:**
Observe the toggle button beside the "Visual Studio Code" in header of the page is not accessible with keyboard when zoomed to 200%.

**Issue repro in below resolutions:**
**Resolution: 1366x768
Resolution: 1280x1024**

**Note:** Same issue observed for all links 

**Expected Result:**
The toggle button should be accessible with keyboard when zoomed for 200%

**Observation:** User may not access menu items "Docs", Updates","Blog","Extensions","FAQ"  ,"SearchDocs", Download".[screenshot]

**User Impact:**
Users with low vision and people who need cognitive assistance will not be able access these interactive elements with keyboard.

**Screenshots:**
![normal view of header items](https://cloud.githubusercontent.com/assets/24825441/21709192/2854e0ea-d405-11e6-941a-db806716b4f2.png)
![toggle button not accessible with 200prcnt zoom](https://cloud.githubusercontent.com/assets/24825441/21709195/2e8e3d80-d405-11e6-8bbf-789fec76b921.png)

**Comment by chrisdias** [comment]
fixed in https://github.com/Microsoft/vscode-website/commit/fa163a42573cd1f45a8229a18090fd06025e2429
